backend/src/hmc-api/fetcher/endpoints.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @author Mia Celeste
+ * @date Feb 2022
+ *
+ * This file is untracked in git and the contents will remain to be a simple mirror
+ * of endpoints-dev. We promised the school to not disclose how exactly we are contacting
+ * their server for data as a security measurement.
```

Comment:
so this file right now is just a placeholder?

Reply:
Yes, and it's gonna forever be left out of version control so we don't leak links and path parameters. Also the amount of settings means it's not quite practical to be env vars.